Makefile

```diff
@@ -2,7 +2,7 @@ NCPUS ?= 1
 
 BASE := logger utils db settings visualization qaqc remote workflow
 
-MODELS := biocro clm45 dalec ed fates gday jules linkages \
+MODELS := clm45 dalec ed fates gday jules linkages \
```

Comment:
I know BioCro has caused a lot of frustration lately, but it's hopefully been fixed now and should not be removed from the Makefile regardless.